mobile/src/src/navigators/TabNavigator.jsx

```diff
@@ -51,26 +52,35 @@ const TabNavigator = ({ navigation }) => {
           ),
         }}
       />
+      
       <Tab.Screen
-        name="Housing"
-        component={HousingPage}
+        name="Create"
+        component={CreatePage}
         options={{
-          tabBarIcon: ({ color }) => (
-            <MaterialCommunityIcons name="home" color={color} size={24} />
+          tabBarIcon: ({ color}) => (
```

Comment:
As you can see, ({ color}) isnt centered. Im going to stop the review now, install the prettier extension and just go through your files youve touched and do ctrl+S, and upload the changes

Reply:
Other than that- i havent tested the code locally yet, so i will do that and let you know. Code looks good though grace, good work.